exercises/alice_russell/ex02_oo_basics/src/Shape.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+class Shape {
```

Comment:
This is a pretty good class definition. It is generally a good idea to avoid prefixing method names with redundant words such as `get` and in the case where methods will just give properties, like these, then just use things like `perimeter()`, `area()`, `sides()`, `type()`.

`print_details` is an example where the qualifier is necessary to understand that there will be a side effect or some computation will probably need to happen. `find_area` could be `calculate_area` if we thought that some significant calculation could be involved.